### PR TITLE
[PYTHON API] fix tensor naming

### PR DESCRIPTION
**Pull request by AlexeyLebedev1**
### Tickets:
 - *72349*

### CODE REVIEW

File: src/bindings/python/src/openvino/ie_api.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def normalize_inputs(py_dict: dict, py_types: dict) -> dict:
 
 def get_input_types(obj: Union[InferRequestBase, ExecutableNetworkBase]) -> dict:
     """Get all precisions from object inputs."""
-    return {i.get_node().get_friendly_name(): i.get_node().get_element_type() for i in obj.inputs}
+    return {i.get_any_name(): i.get_element_type() for i in obj.inputs}
```
Comment (popovaan) [inline src/bindings/python/src/openvino/ie_api.py:41]: Can you add tests for this method? And for infer() method?

File: src/bindings/python/src/pyopenvino/graph/node_output.hpp [hunk 2]
```diff
@@ -41,6 +41,26 @@ void regclass_graph_Output(py::module m, std::string typestring)
                 get_index : int
                     Index value as integer.
                )");
+    output.def("get_any_name",
+               &ov::Output<VT>::get_any_name,
+               R"(
+                Any tensor names associated with this output.
```
Comment (jiwaszki) [inline src/bindings/python/src/pyopenvino/graph/node_output.hpp:47]: Actually I will leave a note here that `get_any_name` return first (in lexicographical order) name associated with the output. In short - this is not "random" function. 
```suggestion
                One of the tensor names associated with this output.
                Note: first name in lexicographical order.
```

File: src/bindings/python/src/pyopenvino/graph/node_output.hpp [hunk 3]
```diff
@@ -41,6 +41,27 @@ void regclass_graph_Output(py::module m, std::string typestring)
                 get_index : int
                     Index value as integer.
                )");
+    output.def("get_any_name",
```
Comment (akuporos) [inline src/bindings/python/src/pyopenvino/graph/node_output.hpp:44]: from my perspective it also would be nice to add properties

File: src/bindings/python/src/openvino/ie_api.py (hunk 1 above)
Comment (AlexeyLebedev1) [reply src/bindings/python/src/openvino/ie_api.py]: done

File: src/bindings/python/src/pyopenvino/graph/node_output.hpp (hunk 3 above)
Comment (AlexeyLebedev1) [reply src/bindings/python/src/pyopenvino/graph/node_output.hpp:44]: done

**Review by akuporos — APPROVED** [review]

**Review by jiwaszki — APPROVED** [review]

**Pull request merged**